spec/index/index_spec.rb

```diff
@@ -28,6 +28,24 @@
       expect(i.to_a).to eq([DateTime.new(2012,2,4), DateTime.new(2012,2,5), DateTime.new(2012,2,6)])
       expect(i.frequency).to eq('D')
     end
+
+    context "create an Index with name" do
+      context "if no name is set" do
+        let(:idx) { Daru::Index.new [:a, :b, :c] }
+        it { expect(idx.name).to be_nil }
```

Comment:
We use the new rspec syntax that uses `its`. Have a look at the other specs for reference.